.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,85 @@
+name: Update node_modules
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, labeled]
```

Comment:
It's not immediately clear to me why you want "opened" **and** labeled? Won't you get two parallel runs then if dependabot opens a PR with labels?

Reply:
Haven't seen that, and we use the same logic for `reposchutz` so this seems to go fine.